spec/models/profile_spec.rb

```diff
@@ -66,6 +63,27 @@
       expect(profile.initiating_service_provider).to be_nil
       expect(profile.verified_at).to be_nil
     end
+
+    context 'with unlinked establishing enrollment' do
+      # When marking profile as pending in-person proofing, the enrollment hasn't yet been linked to
+      # the profile.
+      #
+      # See: Idv::Session#create_profile_from_applicant_with_password
+
+      before do
+        profile.in_person_enrollment.update(status: :establishing, profile: nil)
```

Comment:
When would a Profile be associated with an InPersonEnrollment and the InPersonEnrollment have profile set to nil?

Reply:
Does the comment above help clarify?

https://github.com/18F/identity-idp/blob/d1a1bc77aa5417013e7a16b6559fe7f76476cde6/spec/models/profile_spec.rb#L68-L71

Reply:
maybe clarify that it's still linked to the user via the `in_person_enrollment.user_id`?